src/commands/command.discovery.ts

```diff
@@ -1,5 +1,6 @@
-import { BaseApplicationCommandData } from 'discord.js';
+import { BaseApplicationCommandData, Snowflake } from 'discord.js';
 import { NecordBaseDiscovery } from '../context';
+import { GUILDS_METADATA } from '../necord.constants';
 
 // TODO: Add guild parameter
```

Comment:
Can you remove this TODO before merge PR?

Reply:
Done ✅